projects/owi/programs/owi/module.nix

```diff
@@ -17,5 +17,9 @@ in
     environment.systemPackages = with pkgs; [
       cfg.package
     ];
+
+    demo-shell.owi.programs = {
```

Comment:
We've reworked the demo in https://github.com/ngi-nix/ngipkgs/pull/1403 such that this is set in a separate config, now (also see https://github.com/ngi-nix/ngipkgs/pull/1407).